apps/api/src/user/service/user.service.ts

```diff
@@ -30,7 +30,26 @@ export class UserService {
   }
 
   async getSelf(user: User) {
-    return user
+    let defaultWorkspace = null
+    if (!user.isAdmin) {
+      defaultWorkspace = await this.prisma.workspace.findFirst({
+        where: {
+          ownerId: user.id,
+          isDefault: true
+        },
+        select: {
```

Comment:
You can get rid of the partial select, since we don't have any sensitive content.

Reply:
@unamdev0 can you just fix this one?

Reply:
sorry, missed this, added fix for this now,also fixed package lock in gitignore, please check